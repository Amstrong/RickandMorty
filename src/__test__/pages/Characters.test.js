import React from "react";
import { mount } from "enzyme";
import { Link } from "react-router-dom";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router";

describe("<Characters/>", () => {
  test("Characters: Comprobando Link de ReactRouter", () => {
    const component = renderer.create(
      <StaticRouter location="someLocation" context={{}}>
        <Link to="/" />
      </StaticRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
