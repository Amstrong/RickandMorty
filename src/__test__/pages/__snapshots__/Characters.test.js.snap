// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Characters/> Characters: Comprobando Link de ReactRouter 1`] = `
<a
  href="/"
  onClick={[Function]}
/>
`;
